### DISCUSSION TITLE
Seminario sobre herramientas a utilizar no disponible.

### DISCUSSION THREAD

**Discussion by josebruzzoni** [dds-jv/foro#114] [Consultas de herramientas] [answered]
Buenas, hoy quería ver el seminario que se dicto hace un par de sábados que había quedado grabado en zoom y me aparece el siguiente mensaje: _No se puede ver esta grabación. Sin permiso._
![image](https://user-images.githubusercontent.com/44245325/80324566-cafdca00-8807-11ea-9391-1e52b5c816a4.png)
Ojalá se pueda rehabilitar de alguna forma ya que me vendría muy bien pegarle una reescuchada. 
De todas formas muchas gracias!

**Comment by gastonprieto** [comment]
Buenas!

Tenes el link a mano? porque justo nosotros no manejamos ese seminario y tenemos que averiguar. Por otro lado que estarías buscando ver en particular? porque te recuerdo que tanto el apunte de maven como el de java, tienen linkeados dos videos que muestran varios de los temas que se tocan en ese video.

**Comment by josebruzzoni** [comment, marked as answer]
Ahi parece que lo solucionaron, gracias!!
